libs/angular-integration-interface/src/lib/services/connection.service.ts

```diff
@@ -0,0 +1,28 @@
+import { ENVIRONMENT_INITIALIZER, Injectable, OnDestroy, inject } from '@angular/core'
+import { UserService } from './user.service'
+import { TranslateService } from '@ngx-translate/core'
+import { Subscription } from 'rxjs'
+
+export function provideConnectionService() {
+  return [
+    {
+      provide: ENVIRONMENT_INITIALIZER,
+      multi: true,
+      useFactory() {
+        return () => inject(ConnectionService)
+      },
+    },
+    ConnectionService,
+  ]
+}
+
+@Injectable()
+export class ConnectionService implements OnDestroy {
```

Comment:
Should be in angular-accelerator

Reply:
The whole service should be in angular-accelerator. It has nothing to do with interacting with the shell. angular-integration-interface is the instantiation of integration-interface inside the angular world. Therefore it should only contain services that are instantiating Topics. (We do not have this in all cases but we should not make it worse ;-) )